doc/concepts/pipeline-concepts/pipeline/index.rst

```diff
@@ -0,0 +1,61 @@
+Pipeline
+========
+
+A pipeline is a Pachyderm primitive that is responsible for collecting data
+from a specified source, or ``pfs/input``, transform it according to the pipeline
+configuration file, and place it to the ``pfs/out`` directory in the ``pachd``
+container. Each pipeline subscribes to a branch in the
+input repository or repositories, gets triggered by a commit,
+and runs your code to completion.
+
+A minimum pipeline specification must include the following parameters:
+
+- ``name`` — The name of your data pipeline. You can set an arbitrary
+  name that is meaningful to the code you want to run to process the
+  data in the specified repository.
+- ``input`` — A location of the data that you want to process, such as a
+  Pachyderm repository or a Git repository. You can specify multiple input
```

Comment:
```suggestion
  Pachyderm repository. You can specify multiple input
```
While A git repository is technically correct, git inputs might be going away soon or at least redesigned so let's not mention them here. "such as a pachyderm repo" implies there are other options too, which there are.